meetings/minutes/2023-07-03-TSC-Minutes.md

```diff
@@ -0,0 +1,77 @@
+# Minutes from the 2023-07-03 Laminas Technical Steering Committee Meeting
+
+- When: 2023-07-03 at 19:00 UTC, until 20:20 UTC
+- Where: Laminas Slack #tsc-meeting channel
+- Attending:
+    - Abdul Malik Ikhsan
+    - Alexei Khudiakov
+    - Andreas Heigl
+    - Gary Lockett
+    - George Steel
+    - Luís Cobucci
+    - Marco Pivetta
+    - Max Bösing
+    - Michał Bundyra
+
+Quorum WAS met (9 out of 15 members were present).
```

Comment:
@gsteel 
Interesting:

> 9 out of 15 members were present

Later:

> Ultimately, a vote was cast on whether to first introduce Rector as a scheduled GitHub action to the shared CI workflow which passed **with 10 votes in favour, none against and no abstentions**.

> A vote to reinstate the package as a major dependency of `laminas-cache` passed **with 8 in favour, 2 abstentions and 1 against**.

_By the way, I was there too._

Reply:
Sorry @froschdesign - I didn't see your thumbs-up on the role call :( - I guess that's why there were 10 votes… How we got to 11 on the other vote is anyone's guess